masakari-controller/controller/masakari_worker.py

```diff
@@ -367,34 +345,30 @@ def _skip_process_accident_vm_recovery(self, uuid, vm_state):
             # Call nova stop API.
             self.rc_util_api.do_instance_stop(uuid)
 
-            msg = "[RecoveryControllerWorker_0022]"
-            msg += ("Skipped recovery. instance_id:%s, "
+            msg = ("Skipped recovery. instance_id:%s, "
                     "accident type: [qemu process accident]." % uuid)
-            self.rc_util.syslogout(msg, syslog.LOG_INFO)
+            LOG.info(self.rc_util.msg_with_thread_id(msg))
 
         except EnvironmentError:
-            self.rc_util.syslogout_ex("RecoveryControllerWorker_0023",
-                                      syslog.LOG_ERR)
             status = self.STATUS_ERROR
             error_type, error_value, traceback_ = sys.exc_info()
             tb_list = traceback.format_tb(traceback_)
-            self.rc_util.syslogout(error_type, syslog.LOG_ERR)
-            self.rc_util.syslogout(error_value, syslog.LOG_ERR)
+            LOG.error(self.rc_util.msg_with_thread_id(error_type))
+            LOG.error(self.rc_util.msg_with_thread_id(error_value))
             for tb in tb_list:
-                self.rc_util.syslogout(tb, syslog.LOG_ERR)
+                LOG.error(self.rc_util.msg_with_thread_id(tb))
         except:
-            self.rc_util.syslogout_ex("RecoveryControllerWorker_0024",
-                                      syslog.LOG_ERR)
             status = self.STATUS_ERROR
             error_type, error_value, traceback_ = sys.exc_info()
             tb_list = traceback.format_tb(traceback_)
-            self.rc_util.syslogout(error_type, syslog.LOG_ERR)
-            self.rc_util.syslogout(error_value, syslog.LOG_ERR)
+            LOG.error(self.rc_util.msg_with_thread_id(error_type))
+            LOG.error(self.rc_util.msg_with_thread_id(error_value))
             for tb in tb_list:
-                self.rc_util.syslogout(tb, syslog.LOG_ERR)
+                LOG.error(self.rc_util.msg_with_thread_id(tb))
 
         return status
 
+    @log_process_begin_and_end.output_log
```

Comment:
I have same question about the decorator.  masakari() in masakari_controller.py outputs arguments for the method but the method also is decorated. Do we need same information in log file?